File: snippets/java/array/shuffle/Shuffle.java
package array;
import java.util.Arrays;
import java.util.Random;

public class Shuffle {

    public static void main(String args[]) {
        int[] arr = { 1, 2, 3, 4, 5 };
        Random randNumber = new Random();
        for (int i = 0; i < arr.length; i++) {
            int noToSwap = randNumber.nextInt(arr.length);
            int temp = arr[noToSwap];
            arr[noToSwap] = arr[i];
            arr[i] = temp;
        }
        System.out.println(Arrays.toString(arr));
    }
}

File: snippets/cs/array/shuffle/Shuffle.cs
using System;
using System.Linq;

public class Shuffle
{
    static void Main(string[] args)
    {
        int[] array = { 1, 2, 3, 4, 5 };
        Random randNumber = new Random();
        array = array.OrderBy(x => randNumber.Next()).ToArray();  // ≡
        foreach (int el in array)
        {
            Console.WriteLine(el);
        }
    }
}
